lib/types.js

```diff
@@ -83,7 +94,14 @@ function deepClone(obj) {
   return JSON.parse(JSON.stringify(obj));
 }
 
-function generateTypeConversion(ctx, name, idlType, argAttrs = [], parentName, errPrefix = '"The provided value"') {
+function generateTypeConversion(
+  ctx,
+  name,
+  idlType,
+  argAttrs = [],
+  parentName = undefined,
+  errPrefix = '"The provided value"'
```

Comment:
I'm personally a fan of trailing commas too

Reply:
I can't bring myself to do it yet...